src/ocean/text/utf/UtfUtil.d

```diff
@@ -34,13 +34,10 @@
 
 module ocean.text.utf.UtfUtil;
 
-
```

Comment:
It'd really help review if you didn't meld whitespace changes in with other types of change.